libs/ui-watt/src/lib/foundations/typography/_typography.import.scss

```diff
@@ -14,97 +14,155 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@use 'sass:map';
+
+@use '../breakpoints/breakpoints.import' as breakpoints;
+@use '../../utils/rem' as utils;
+
+$primary-font-family: 'Open Sans', sans-serif;
+$secondary-font-family: 'Open Sans', sans-serif;
+
+$font-weights: (
+  'light': 300,
+  'regular': 400,
+  'semi-bold': 600,
+  'bold': 700,
+);
+
+$font-sizes: (
+  'xxl': (
+    'font-size': 32px,
+    'line-height': 42px,
+  ),
+  'xl': (
+    'font-size': 26px,
+    'line-height': 36px,
+  ),
+  'l': (
+    'font-size': 20px,
+    'line-height': 28px,
+  ),
+  'm': (
+    'font-size': 16px,
+    'line-height': 24px,
+  ),
+  's': (
+    'font-size': 14px,
+    'line-height': 20px,
+  ),
+);
+
+@mixin font-size($size) {
+  $sizes: map.get($font-sizes, $size);
+  font-size: utils.rem(map.get($sizes, 'font-size'));
+  line-height: utils.rem(map.get($sizes, 'line-height'));
+}
+
+@mixin font-weight($weight) {
+  font-weight: map.get($font-weights, $weight);
+}
+
 @mixin watt-headline-1 {
   color: var(--watt-color-primary-dark);
```

Comment:
question: One of the pain points related to typography that we heard from team Fenris was that typography mixins contain color declarations. So they had to overwrite the color declaration in many places. Was it considered moving the color declarations out of the mixins?

Reply:
color has been removed from all typography styles where we don't want a default (this is aligned with Kenneth). I've added new css-vars to make it easy to overwrite these defaults ex. 

`:root {
  --watt-typography-headline-color: #my-awesome-color;
}`